docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,318 @@
+---
+# required/recommended metadata
+
+title: Datetime and DateTimeOffset Support in System.Text.Json
+description: An overview of how Datetime and DateTimeOffset types are supported in the System.Text.Json library.
+author: layomia
+ms.author: laakinri
+ms.date: 07/22/2019
+ms.topic: [TOPIC TYPE]
+ms.prod: [PRODUCT VALUE]
+helpviewer_keywords:
+  - "JSON DateTime JSON DateTimeOffset"
+  - "Serializer, Utf8"
+  - "JSON Reader, JSON Writer, times"
+
+---
+# Datetime and DateTimeOffset Support in System.Text.Json
+
+The <xref:System.Text.Json.JsonSerializer>, <xref:System.Text.Json.Utf8JsonReader>, <xref:System.Text.Json.Utf8JsonWriter>, and <xref:System.Text.Json.JsonElement> parse and write <xref:System.DateTime> and <xref:System.DateTimeOffset> text representations according to an extended profile of the ISO 8601-1:2019 format.
```

Comment:
I think this document is currently too abstract and spec heavy. I'm sure there are some people out there that care about the exact ISO 8601 specification but they're a minority. When you say it supports the ISO 8601 format you should then give a nice example: `2008-09-15T15:53:00`.

In this section you should now have code examples. Extremely basic scenarios of working with ISO 8601 text. Serializing it, parsing it, writing it.